dc/s2n-quic-dc/src/packet/tag.rs

```diff
@@ -79,7 +79,7 @@ decoder_value!(
                     Ok((Self::UnknownPathSecret(tag), buffer))
                 }
                 // reserve this range for other packet types
-                0b0110_0000..=0b0111_1111 => Err(s2n_codec::DecoderError::InvariantViolation(
+                0b0110_0011..=0b0111_1111 => Err(s2n_codec::DecoderError::InvariantViolation(
```

Comment:
The largest value accounted for here is REPLAY_DETECTED:
https://github.com/aws/s2n-quic/blob/001617859c1aaa6744d0ad3ee5bde8134438fc0a/dc/s2n-quic-dc/src/packet/secret_control.rs#L19